integration/kotlinx-coroutines-play-services/build.gradle

```diff
@@ -0,0 +1,82 @@
+import java.nio.file.Files
+import java.nio.file.NoSuchFileException
+import java.util.zip.ZipEntry
+import java.util.zip.ZipFile
+
+/*
+ * Copyright 2016-2018 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+ext.tasks_version = '15.0.1'
+
+repositories {
+    google()
+}
+
+def attr = Attribute.of("artifactType", String.class)
+configurations {
+    aar {
+        attributes { attribute(attr, ArtifactTypeDefinition.JAR_TYPE) }
+        sourceSets.main.compileClasspath += it
+        sourceSets.test.compileClasspath += it
+        sourceSets.test.runtimeClasspath += it
+    }
+}
+
+dependencies {
+    registerTransform {
+        from.attribute(attr, "aar")
+        to.attribute(attr, "jar")
+        artifactTransform(ExtractJars.class)
+    }
+
+    aar("com.google.android.gms:play-services-tasks:$tasks_version") {
```

Comment:
The build won't understand it because the dependency is only available as an aar. That's why the other PR had Android related stuff and the Android Gradle Plugin. Here, we manually extract extract files we care about from the aar.

Reply:
thanks for the explanation